templates/concert-ticket/concert-ticket.json

```diff
@@ -0,0 +1,44 @@
+{
+    "ticket_number": "000385724",
+    "event_time": "2021-05-25T19:30:00.00Z",
+    "ticket_detail": [
+        {
+            "field_name": "Admission",
+            "field_value": "General admission"
+        },
+        {
+            "field_name": "Ticket type",
+            "field_value": "Adult"
+        },
+        {
+            "field_name": "Price",
+            "field_value": "$32"
+        },
+        {
+            "field_name": "Name",
+            "field_value": "Coleman R. Merrell"
+        },
+        {
+            "field_name": "address",
```

Comment:
Probably name this field `Address`, with a capital `A` so that it appears in the output as the other fields.